src/lang/modifyAst/boolean.ts

```diff
@@ -0,0 +1,345 @@
+import { Node } from '@rust/kcl-lib/bindings/Node'
```

Comment:
This file is not very DRY, it's possible that something more abstract could have been done for all three cases, but it's likely that three cases start to diverge as we make the CSG more advanced and we add more kwargs to these. And I think the abstraction would start to get in the way, so each case has it's own function.